pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -623,79 +620,127 @@ def copyTurbineFilesForEachCase(self, writeFiles=True):
+                        # y_rot = x1*sind(-self.inflow_deg[case]) + y1*cosd(-self.inflow_deg[case])
                         if writeFiles:
                             if t==0: shutilcopy2_untilSuccessful(os.path.join(self.templatePath,self.bladefilename), os.path.join(currPath,self.bladefilename))
                             if t==0: shutilcopy2_untilSuccessful(os.path.join(self.templatePath,self.towerfilename), os.path.join(currPath,self.towerfilename))
                             self.ElastoDynFile.write(os.path.join(currPath,f'{self.EDfilename}{t+1}_mod.dat'))
+                            if self.floating == 1:
+                                self.HydroDynFile['PotFile'] = os.path.join(self.templatePath,'HydroData', f'Activefloat_T{t+1}')
+                                self.HydroDynFile.write(os.path.join(currPath, f'{self.HDfilename}{t+1}_mod.dat'))
+                                self.MoorDynFile     = self._check_and_open(self.MDfilepath+f'{t+1}.dat')
+                                for line in range(len(self.MoorDynFile['ConnectionProp'])):
+                                    x1=float(self.MoorDynFile['ConnectionProp'][line][2])
+                                    y1=float(self.MoorDynFile['ConnectionProp'][line][3])
+                                    x_rot = x1*cosd(self.inflow_deg[case]) + y1*sind(self.inflow_deg[case])    
+                                    y_rot = -x1*sind(self.inflow_deg[case]) + y1*cosd(self.inflow_deg[case])
+                                    self.MoorDynFile['ConnectionProp'][line][2]= str(round(x_rot,2))
+                                    self.MoorDynFile['ConnectionProp'][line][3]= str(round(y_rot,2))
+                                self.MoorDynFile.write(os.path.join(currPath, f'{self.MDfilename}{t+1}_mod.dat'))
         
                     elif EDmodel_ == 'SED':
                         # Update each turbine's Simplified ElastoDyn
-                        self.SElastoDynFile['RotSpeed']  = self.bins.sel(wspd=Vhub_, method='nearest').RotSpeed.values
-                        self.SElastoDynFile['BlPitch']   = self.bins.sel(wspd=Vhub_, method='nearest').BlPitch.values
-
                         self.SElastoDynFile['BlPitch']  = self.bins.sel(wspd=Vhub_, method='nearest').BlPitch.values
                         self.SElastoDynFile['RotSpeed'] = self.bins.sel(wspd=Vhub_, method='nearest').RotSpeed.values
                         self.SElastoDynFile['NacYaw']   = yaw_deg_ + yaw_mis_deg_
                         if writeFiles:
                             self.SElastoDynFile.write(os.path.join(currPath,f'{self.SEDfilename}{t+1}_mod.dat'))
-        
+                            
                     # Update each turbine's ServoDyn
                     self.ServoDynFile['YawNeut']      = yaw_deg_ + yaw_mis_deg_
-                    self.ServoDynFile['DLL_FileName'] = f'"{self.DLLfilepath}{t+1}.so"'
+                    self.ServoDynFile['DLL_FileName'] = f'"{self.DLLfilepath}{t+1}.dll"'
```

Comment:
`so` are used in *nix and `dll` in Windows. We need to identify what system the user is using or not specify the extension.

Reply:
yes I agree